guides/source/configuring.md

```diff
@@ -1127,7 +1127,14 @@ application name.
 
 #### `config.active_record.query_log_tags_format`
 
-A `Symbol` specifying the formatter to use for tags. Valid values are `:sqlcommenter` and `:legacy`.  Defaults to `:legacy`.
+A `Symbol` specifying the formatter to use for tags. Valid values are `:sqlcommenter` and `:legacy`.
+
+The default value depends on the `config.load_defaults` target version:
+
+| Starting with version | The default value is |
+| --------------------- | -------------------- |
+| 7.1                   | `:legacy`            |
+| 7.2                   | `sqlcommenter`       |
```

Comment:
Since this is technically a breaking API change, do we know which version of Rails should include it?

Based on the [Rails maintenance policy](https://guides.rubyonrails.org/maintenance_policy.html), a minor version release "may contain API changes (Serve as major versions of Semver)", and a major version release "will likely contain API changes" as well. So I'm a little unsure whether we want update this default value in Rails 7.2, or 8.0.

I noticed 7.2 was referenced for a few other api changes, so I used that for now.
But I'm open to whatever the core team members will recommend :)